postcss.config.js

```diff
@@ -16,32 +18,35 @@ let reactCalendarPath = path.resolve(
 );
 
 if (isVedaInstance) {
-  basePath = vedaPath;
+  vedaUIPath = vedaPath;
   uswdsPath = path.resolve(
-    basePath,
+    vedaUIPath,
     'node_modules/@trussworks/react-uswds/lib'
   );
   reactCalendarPath = path.resolve(
-    basePath,
+    vedaUIPath,
     'node_modules/react-calendar/dist/**/*.css'
   );
+
+  contentPaths = [
+    `${vedaUIPath}/app/**/*.{js,jsx,ts,tsx, html}`,
+    `${basePath}/overrides/**/*.{js,jsx,ts,tsx}`
```

Comment:
should this also include `custom-pages`? (sorry if I'm misunderstanding this :D )

Reply:
Yes, I forgot that there are also custom pages; I will add them. Thanks for catching it!